common/config/rush/browser-approved-packages.json

```diff
@@ -18,6 +18,10 @@
       "name": "axios",
       "allowedCategories": [ "libraries" ]
     },
+    {
+      "name": "chalk",
+      "allowedCategories": [ "libraries" ]
+    },
```

Comment:
Use `colors` (imported as `colors/safe`), but this should go in `nonbrowser-approved-packages.json`

Reply:
Or, you can use `Terminal` from `node-core-library`, and use its `Colors` functions.

Reply:
+1

Reply:
changed to use colors/safe